src/masoniteorm/connections/MSSQLConnection.py

```diff
@@ -0,0 +1,146 @@
+import random
+
+from ..exceptions import DriverNotFound
+from .BaseConnection import BaseConnection
+from ..query.grammars import MSSQLGrammar
+from ..schema.platforms import MSSQLPlatform
+
+
+CONNECTION_POOL = []
+
+
+class MSSQLConnection(BaseConnection):
+    """Postgres Connection class."""
```

Comment:
Don't forget to update the docstring here
```suggestion
    """MSSQL Connection class."""
```